template.yml

```diff
@@ -1195,20 +1129,24 @@ Resources:
               Action:
                 - sts:AssumeRole
       ManagedPolicyArns:
+        - !Ref ArtifactBucketReadOnlyAccessPolicy
+        # For uploading files and file parts to the artifact store
+        - !Ref ArtifactBucketWriteAccessPolicy
         - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
         - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
-      Path: /
       Policies:
-        - PolicyName: ArtifactBucketWriteAccess
-          PolicyDocument:
-            Version: "2012-10-17"
-            Statement:
-              - Effect: Allow
-                Action:
-                  - s3:Get*
-                  - s3:Put*
-                Resource:
-                  - !Sub ${ArtifactBucket.Arn}/*
+        # TODO Need to make sure the ArtifactBucketWriteAccessPolicy policy
```

Comment:
So AWS transcribe might need read and/or list access to the destination?

Reply:
I can't remember if I gave it `put*` and `get*` because it needed it, or because I was being lazy